e2e/modeling/test_adaptive_qa_inference.py

```diff
@@ -1,9 +1,11 @@
 import pytest
 
+from haystack.modeling.infer import Inferencer
+
 
 @pytest.mark.parametrize("multiprocessing_chunksize", [None, 2])
-@pytest.mark.parametrize("num_processes", [2, 0, None], scope="module")
-def test_qa_format_and_results(adaptive_model_qa, multiprocessing_chunksize):
+@pytest.mark.parametrize("num_processes", [2, 0, None])
+def test_qa_format_and_results(multiprocessing_chunksize, num_processes):
```

Comment:
num_processes is not used anymore; let's just remove it here and shorten the test runtime

Reply:
@ZanSara wait has num_processes been addressed? Why not remove it altogether, as we don't use it in inferencer at all and if I am not mistaken we run this test 3x more for no reason :-)